west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: v2.4.0-ncs1-rc1
+      revision: pull/369/head
```

Comment:
Update to 1d4bbe5b7695f003b823c4d307706efdd4a44415

Reply:
Actually, wait a bit. We want to merge other PRs first right @tejlmand ?

Reply:
> 
> 
> Actually, wait a bit. We want to merge other PRs first right @tejlmand ?

If we could get https://github.com/nrfconnect/sdk-zephyr/pull/373 in also, we could make a single manifest update, and safe some CI rounds.
And if https://github.com/nrfconnect/sdk-zephyr/pull/376 could join, it would safe even one more round.

Reply:
We could also include: https://github.com/nrfconnect/sdk-zephyr/pull/374
it's CI/manifest run is green: https://github.com/nrfconnect/sdk-nrf/pull/3158

also the changes looks unrelated to the other sdk-zephyr PRs, so should be safe to join them in single manifest update.

Reply:
and https://github.com/nrfconnect/sdk-zephyr/pull/377

Reply:
This is blocking our other tasks. It is waiting to be merged for too long!

Reply:
@tejlmand Here the sdk-zephyr PR is already merged and other PRs are still open. Can we update SHA and merge here first?